templates/sponsorship-pledge.html

```diff
@@ -0,0 +1,59 @@
+    </li> 
+    <li>
+        <b>Bevy leaders must be open about who supports them (within the context of Bevy)</b>. No hidden entities behind the scenes. If we receive $2,400 a year or more from an entity or have a financial stake in one that relates to the Bevy project in any reasonable way, we must disclose it within 6 months of receiving the money or stake.
+    </li>
+    <li>
+        <b>Bevy leaders will hold each other accountable</b>. If any of us suspects a violation of the pledge, we will immediately seek to rectify it. First amongst ourselves if there is reasonable doubt, and then publicly if the situation demands it.
+    </li>
+    <li>
+        <b>The pledge and the Bevy Community come first ... always</b>. If we are confronted with the choice of upholding the pledge or losing the financial support of an entity, we will choose the pledge every time. The amount of money on the table has no bearing on this. We must be honest with ourselves and each other. If we can't say no to large sums of money with pledge-violating strings attached, we don't deserve to be Bevy leaders.
+    </li>
+</ol>
+
+<p>
+The Bevy project (and the financial support we receive) is constantly growing. We must be certain we don't lose ourselves, our purpose, or the trust of the community as we continue on this wild ride. This is also a living document. If you believe we missed something important, please let us know. Removing items from the list would almost certainly be a violation of The Pledge, so unless we've seriously messed something up, consider this append-only.
+</p>
+
+<p>
+Currently, this agreement operates on the honor system. We believe all of our maintainers have won the trust of the community, but we will be putting together a legal organization to uphold this agreement and help fairly collect and distribute funds later this year.
+</p>
+
+<p>
+Also note that this only applies to Bevy leadership. We welcome and encourage companies to fund development efforts that improve Bevy in whatever way they want. However Bevy leadership <i>cannot be involved in any way that violates The Pledge</i>. Bevy leadership must always be the filter that protects the community from changes that go against their interests. Companies should do their best to collaborate with the Bevy community (and Bevy leaders) to ensure their changes are acceptable.
+</p>
+
+<h2>Pledge Signers</h2>
+<ul>
+    <li>
+        <a href="https://github.com/sponsors/cart">Carter Anderson (@cart)</a> 
+        <ul>
+            <li>Work Status: Full-time Project Lead / Maintainer</li>
+            <li>Sponsors (above $2,400 / year threshold): Foresight Mining Software Corporation ($2,000 / month), Embark ($1,000 / month), MeetKai ($1,000 / month), Encultured ($1,000 / month), Striked ($200 / month), storytold ($200 / month), Metabuild ($100 / month), Keygen ($100 / month), VPSServer ($100 / month), @joshuajbouw ($200 / month), @CleanCut ($200 / month), @rozgo ($100 / month)</li>
```

Comment:
```suggestion
            <li>Sponsors (including all those  above $2,400 / year threshold): Foresight Mining Software Corporation ($2,000 / month), Embark ($1,000 / month), MeetKai ($1,000 / month), Encultured ($1,000 / month), Striked ($200 / month), storytold ($200 / month), Metabuild ($100 / month), Keygen ($100 / month), VPSServer ($100 / month), @joshuajbouw ($200 / month), @CleanCut ($200 / month), @rozgo ($100 / month)</li>
```

Reply:
There are $100 / month sponsors on here, that I think are probably worth including.